lib/pool_connection.js

```diff
@@ -42,9 +43,9 @@ class PoolConnection extends Connection {
     this.release();
   }
 
-  destroy() {
+  destroy(...args) {
```

Comment:
this fails in node 4

Reply:
Aww snap, sorry I've been testing on v8 locally 😛

Reply:
I'm on v10 locally :)